sdk/core/azure-core/test/ut/transport_adapter.cpp

```diff
@@ -14,7 +14,7 @@ namespace Azure { namespace Core { namespace Test {
   {
     std::vector<std::unique_ptr<Azure::Core::Http::HttpPolicy>> p;
     std::shared_ptr<Azure::Core::Http::HttpTransport> transport
-        = std::make_shared<Azure::Core::Http::CurlTransport>();
+        = std::make_shared<Azure::Core::Http::WinHttpTransport>();
```

Comment:
Nice to have, Would be nice if we used a set w/ CurlTransport and WinHttpTransport then ran all the transport tests with those two.  wouldn't be feasible for every test, but knowing that adding a test for one also validated the other would be really nice

Reply:
Yep, that's the plan.

Reply:
Fixed in this current PR and in https://github.com/Azure/azure-sdk-for-cpp/pull/928.